gemfiles/4.2.gemfile.lock

```diff
@@ -0,0 +1,56 @@
+PATH
```

Comment:
I would not commit `Gemfile.lock`  since I always want to test the newest version of gems specify in Gemfile.